deli_counter.rb

```diff
@@ -0,0 +1,46 @@
+# Anshul Arora
+katz_deli = []
+$current_line = []
+
+def line(arr)
+  if arr.empty?
+    puts "The line is currently empty." 
+  else
+    $current_line.clear
+    line_helper(arr)
+    puts "The line is currently:#{$current_line.join}"
+  end
+end
+
+
+def line_helper(arr)
+  arr.each do |name|
```

Comment:
make use of `each_with_index()` so you don't need to define a position by making use of `index()` inside a loop. this is much more inefficient than simply using `each_with_index()`. `each` gets called `n` times, then inside of this loop, `index` gets called running `n` times as well - this totals to an `n^2` runtime. Simply using `each_with_index()` would do both of these operation in one loop, resulting in a much better linear (`n`) runtime

Reply:
also `customer` might be a better  variable name over `name`